saleor/graphql/translations/schema.py

```diff
@@ -88,3 +131,35 @@ def resolve_translations(self, info, kind, **_kwargs):
             return resolve_menu_items(info, query=None)
         elif kind == TranslatableKinds.SALE:
             return resolve_sales(info, query=None)
+
+    @permission_required("site.manage_translations")
+    def resolve_translation(self, info, id, kind, **_kwargs):
```

Comment:
You don't need to do so many `return` options.
How about this:
a) 
```
if kind == TranslatableKinds.PRODUCT:
    model = Product
elif ....
...
return model.objects.filter(pk=kind_id).first()
```
or (I like this one :D )
b)
```
kinds = {TranslatableKinds.PRODUCT: Product, ..., TranslatableKinds.COLLECTION:Collection}
model = kinds[kind]
model.objects.filter(pk=kind_id).first()
```